automerge/src/autotxn.rs

```diff
@@ -0,0 +1,417 @@
+use crate::exid::ExId;
+use crate::transaction::CommitOptions;
+use crate::types::Patch;
+use crate::{
+    change::export_change, transaction::TransactionInner, ActorId, Automerge, AutomergeError,
+    Change, ChangeHash, Prop, Value,
+};
+use crate::{SyncMessage, SyncState};
+
+/// An automerge document that automatically manages transactions.
+#[derive(Debug, Clone)]
+pub struct AutoTxn {
```

Comment:
I wonder if this might be better called `AutoCommit` to evoke autocommit options on various SQL database adapters? I don't feel strongly about this, just a thought.

Reply:
Yes! That makes much more sense.